qupath-gui-fx/src/main/java/qupath/lib/gui/JavadocViewerRunner.java

```diff
@@ -89,6 +91,8 @@ public JavadocViewerRunner(Stage owner) {
                         .filter(Objects::nonNull)
                         .toArray(URI[]::new)
         );
+        if (!command.getJavadocViewer().getStyleClass().contains(JAVADOCS_STYLE_CLASS))
+            command.getJavadocViewer().getStyleClass().add("javadoc-viewer");
```

Comment:
I think it would be better to put that part in the `run()` function.

When you instantiate a `JavadocViewerCommand`, the underlying `JavadocViewer` is not created yet. However, calling `command.getJavadocViewer()` will create the `JavadocViewer` (and proceed to search for Javadocs).

As the `JavadocViewerRunner` is instantiated during QuPath startup, I think it's better if we delay the `JavadocViewer` creation to the moment when we actually open the `JavadocViewer` window for the first time. This may increase QuPath startup time.